.github/workflows/pull.yml

```diff
@@ -110,3 +110,21 @@ jobs:
     uses: ./.github/workflows/_unittest.yml
     with:
       docker-image: executorch-ubuntu-22.04-clang12
+
+  test-arm-backend-delegation:
```

Comment:
Are we running this on PRs? I think we should run all vendor-specific jobs ins trunk but with the ability to run it on PRs with labeling.

Reply:
Yes. I will remove this part. It's testing only.